code/modules/surgery/organs/organ_internal.dm

```diff
@@ -98,13 +98,16 @@
 
 //Checks to see if the organ is frozen from temperature
 /obj/item/organ/proc/is_cold()
-	var/freezing_objects = list(/obj/structure/closet/crate/freezer, /obj/structure/closet/secure_closet/freezer, /obj/structure/bodycontainer, /obj/item/autosurgeon)
+	var/freezing_objects = list(/obj/structure/closet/crate/freezer, /obj/structure/closet/secure_closet/freezer, /obj/structure/bodycontainer, /obj/item/autosurgeon, /obj/machinery/smartfridge/organ)
 	if(istype(loc, /obj/))//Freezer of some kind, I hope.
 		if(is_type_in_list(loc, freezing_objects))
 			if(!(organ_flags & ORGAN_FROZEN))//Incase someone puts them in when cold, but they warm up inside of the thing. (i.e. they have the flag, the thing turns it off, this rights it.)
 				organ_flags |= ORGAN_FROZEN
 			return TRUE
-		return
+		var/external_check = FALSE
+		if(organ_flags & ORGAN_FROZEN)
+			external_check = TRUE
+		return external_check
```

Comment:
```
		return (organ_flags & ORGAN_FROZEN)
```

Reply:
Ahh I wanted to do that but didn't realise putting it in () would let it return true/false, neat!